frontend/src/app/modules/fields/edit/field-types/date-edit-field.component.ts

```diff
@@ -26,19 +26,16 @@
 // See docs/COPYRIGHT.rdoc for more details.
 // ++
 
-import {Component} from "@angular/core";
+import {Component, OnInit} from "@angular/core";
 import * as moment from "moment";
 import {TimezoneService} from "core-components/datetime/timezone.service";
 import {EditFieldComponent} from "core-app/modules/fields/edit/edit-field.component";
 import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
+import {DatePickerModal} from "core-components/datepicker/datepicker.modal";
+import {OpModalService} from "core-components/op-modals/op-modal.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/8ce9d25d81c89fa5502fed98ea5e9dbaa1292905/frontend/src/app/modules/fields/edit/field-types/date-edit-field.component.ts#L36"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>